notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb

```diff
@@ -525,7 +525,7 @@
         "        f.write('{\"id\":\"' + str(i) + '\",')\n",
```

Comment:
<p>combine the three code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Which three code cells are you referring to here?